packages/shopping-cart/src/types.ts

```diff
@@ -419,16 +428,13 @@ export interface ResponseCartProduct {
 	 *
 	 * Note that the difference may be caused by many factors, not just coupons.
 	 * It's best not to rely on it.
-	 *
 	 * @deprecated This is a float and is unreliable. Use coupon_savings_integer
 	 */
 	coupon_savings?: number;
 
 	/**
-	 * The difference between `cost_before_coupon` and the actual price in the currency's smallest unit.
-	 *
-	 * Note that the difference may be caused by many factors, not just coupons.
```

Comment:
I've tried to make this property more accurate in D129313-code so we can start relying on it.